docs/development/testing_configuration_tests.rst

```diff
@@ -80,23 +80,24 @@ relevant location for the host you plan to test: ::
     └── test_ossec.py
 
 In the example above, to add a new test for the ``app-staging`` host,
-add a new file to the ``testinfra/spec/app-staging`` directory.
+add a new file to the ``testinfra/staging/app`` directory.
 
 .. tip:: Read :ref:`updating_ossec_rules` to learn how to write tests for the
          OSSEC rules.
 
 Config Test Layout
 ------------------
 
-The config tests are mostly broken up according to machines in the
-Vagrantfile: ::
+With some exceptions, the config tests are broken up according to platform definitions in the
+Molecule configuration: ::
 
-    testinfra/
+    molecule/testinfra/staging
     ├── app
     ├── app-code
-    ├── build
     ├── common
-    └── mon
+    ├── mon 
+    ├── ossec
+    └── vars
```

Comment:
These path updates are long overdue, thanks for fixing! :ok_hand:

Reply:
:metal: